### DISCUSSION TITLE
[Feature]: Charts to follow the size of other elements

### DISCUSSION THREAD

**Discussion by AmdyTran** [evidence-dev/evidence#2421] [General]
### Description

It would be nice to be able to create a grid of for example a Data Table & Chart, 
Then have the chart automatically fill up the complete size of the grid, especially if the Data Table is much bigger.

![image](https://github.com/evidence-dev/evidence/assets/55977578/e1084206-3448-40ef-afd5-00499e23ced1)

Idea could be like this, where instead LHS could be a very long table 

### How should it work?

In the component of a chart for example we can specify the maxHeight = 100% for example, to take up the whole spacing, or height= auto to take up the full height.

**Comment by ItsMeBrianD** [comment]
Because we are using css grid under the hood, this may be a complex one to implement, but I can think of a workaround that may suit your use case:

```html
<Grid cols=2>
    <Grid cols=1> <!-- this might not be needed, could be a `<div>` instead? -->
        <Chart/>
        <Chart/>
    </Grid>
    <Chart height="100%"/>
</Grid>
```
